examples/server/server.c

```diff
@@ -2498,32 +2682,47 @@ THREAD_RETURN WOLFSSL_THREAD server_test(void* args)
 
 #ifdef WOLFSSL_DTLS
         if (doDTLS && dtlsUDP) {
-            SOCKADDR_IN_T cliaddr;
             byte          b[1500];
             int           n;
-            socklen_t     len = sizeof(cliaddr);
+
+            client_len = sizeof client_addr;
 
             /* For DTLS, peek at the next datagram so we can get the client's
              * address and set it into the ssl object later to generate the
              * cookie. */
             n = (int)recvfrom(clientfd, (char*)b, sizeof(b), MSG_PEEK,
-                              (struct sockaddr*)&cliaddr, &len);
+                              (struct sockaddr*)&client_addr, &client_len);
             if (n <= 0)
                 err_sys_ex(runWithErrors, "recvfrom failed");
 
             if (doBlockSeq) {
-                XMEMCPY(&dtlsCtx.peer.sa, &cliaddr, len);
-                dtlsCtx.peer.sz = len;
+                XMEMCPY(&dtlsCtx.peer.sa, &client_addr, client_len);
+                dtlsCtx.peer.sz = client_len;
                 dtlsCtx.wfd = clientfd;
                 dtlsCtx.failOnce = 1;
 
                 wolfSSL_SetIOWriteCtx(ssl, &dtlsCtx);
             }
             else {
-                wolfSSL_dtls_set_peer(ssl, &cliaddr, len);
+                wolfSSL_dtls_set_peer(ssl, &client_addr, client_len);
             }
         }
 #endif
+
+#ifdef WOLFSSL_WOLFSENTRY_HOOKS
```

Comment:
Please format the lines to avoid longer than 80 chars. At 2721

Reply:
:heavy_check_mark: